client/src/utils/fuzzyDateOffset.js

```diff
@@ -1,4 +1,12 @@
 export const fuzzyDateOffset = (datestring) => {
+    if (!datestring) return "Never Updated";
+
+    // Some browsers (firefox, safari) cannot parse datestrings of the following form:
+    // 2021-02-11 16:02:12.877406+00
+    if (datestring.includes(" ")) {
+        datestring = datestring.replace(' ','T').replace('+00', 'Z')
```

Comment:
This kind of date manipulation raises red flags for me right off the bat.

You can **change** your browser date format, as I understand it this is based on your language in Chrome and Firefox and a separate setting in Safari.

There must be a lib that can help us out here. I'm convinced this is brittle. Open to being persuaded otherwise but dates always set off alarm bells for me.

Reply:
Okay, I can see where you're coming from on that front. I'll do a bit more digging into a better solution than string manipulation.